apps/web-giddh/src/app/vouchers/utility/vouchers.store.ts

```diff
@@ -1416,4 +1435,142 @@ export class VoucherComponentStore extends ComponentStore<VoucherState> {
         );
     });
 
+    readonly poStatusUpdate = this.effect((data: Observable<{ accountUniqueName: string, payload: any }>) => {
```

Comment:
add documentation

Reply:
There is not a single function where documentation is written in store, that's why I did not write,

please approve

Reply:
If you are adding new method, then you have to write documentation.